recipes/warp-lang/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "warp-lang" %}
+{% set version = "0.10.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIA/warp/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: fc80bc6c38ac2e0a1a59f9dc912a29692bcf3db2848625890fbcdac45b2c06c9
+  patches:
+    - 01-find_g++_env.patch
+    - 02-add_macro_def_build.patch
+    - 03-install_deps_llvm_conda.patch
+    
+build:
+  script: 
+    - {{ PYTHON }} build_lib.py
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation --no-cache-dir  
+  number: 0
+  skip: true  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - gitpython
+    - cmake
+    - ninja
+    - numpy
+  run:
+    - python
+
+test:
+  imports:
+    - warp
+  commands:
+    - pip check
+    # Python tests require CUDA 11.5
+    # - {{ PYTHON }} -m warp.test 
+  requires:
+    - pip
+    - pytest
+    
+about:
+  home: https://github.com/NVIDIA/warp
+  summary: 'A Python framework for high-performance simulation and graphics programming.'
+  license: BSD-4-Clause AND BSD-3-Clause
```

Comment:
The license of warp is not listed here, for example it contains a non-commercial clause. Probably we should add a custom `LicenseRef-` SPDX license. Probably `LicenseRef-NVIDIA-warp` could make sense in this context?